homeassistant/components/escea/translations/bg.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
No need to include translated files here, they will be exported automatically from lokalize.

Reply:
Removed all but strings, en and es